Geometry/HGCalGeometry/src/HGCalGeometry.cc

```diff
@@ -305,7 +305,7 @@ HGCalGeometry::CornersVec HGCalGeometry::getCorners(const DetId& detid) const {
 						id.iCell1,id.iCell2,true,false);
     }
     float dx = m_cellVec[cellIndex].param()[FlatHexagon::k_r];
-    float dy = m_cellVec[cellIndex].param()[FlatHexagon::k_R];
+    float dy = 0.5*m_cellVec[cellIndex].param()[FlatHexagon::k_R];
```

Comment:
Should a constant with an explanatory name be used here instead of the raw number?

Reply:
Is 0.5 a common constant. Should I introduce half as a constant with value 0.5?